app/types.tsx

```diff
@@ -6,3 +6,14 @@ export type Store = {
 };
 
 export type Size = "large" | "small";
+
+export type DealAndStore = Deal & { store: Store };
```

Comment:
`DealAndStore` kinda sounds like the type would be `Deal & Store`, probably more like `DealWithStore`

Reply:
https://github.com/auberginearms/groceryDatabaseApp/pull/26/commits/e0eccbfc1a04ae204b6f045e19ce39350d9ebe08